### PR TITLE
`Development`: Fix typos in code comments

### PR DESCRIPTION
**Pull request by Strohgelaender**
#### General
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context
This PR fixes a multitude of typos or grammar mistakes around the whole code base. I used the integrated IntelliJ-Inspection to spot theses issues. 

### Description
These are a few noteworthy changes:
- Use the offical spelling for brand names (`Github` -> `GitHub`, `MacOS` -> `macOS`)
- Add hyphens where needed (`semi automatic` -> `semi-automatic`, `non existing` -> `non-existing`)
- `login` -> `log in` if used as a verb
- Fix a / an and its / it's mistakes

### Steps for Testing
Code Review

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/7a56396dabe78c790261ae5dcd93b6880ba514d4/src/test/java/de/tum/in/www1/artemis/AssessmentTeamComplaintIntegrationTest.java#L206

Result, but this would require a bit of refactoring as this is a variable name

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/artemis-server-cli#L6
Common

https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/artemis-server-cli#L7
assumes

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/LICENSE#L17
Non infringement (2 words, not 1)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/docs/dev/setup/common-problems.rst#L51

Retrieve

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/.github/workflows/build-deploy.yml#L175

Download

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/src/main/webapp/app/shared/chart/chart-exercise-type-filter.directive.ts#L34

exerciseScores

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/b3d021b6d9bb45fd00845fed00fe7e8144554684/src/main/webapp/app/shared/markdown-editor/commands/colorPicker.command.ts#L14
Color change occurs? Not sure what was initially meant

**Comment by Strohgelaender** [comment]
Thank you for the review @babyygemperor. I changed everything except one:

> NONINFRINGEMENT

I won't change this since it is part of the licence (and I expect it to be copied from somewhere)

**Review by Marvin-Vitcu — CHANGES_REQUESTED** [review]
TIP FOR REVIEWERS: USE UNIFIED VIEW

reviewed all of the changes. Found one spelling mistake:

bee --> be
image
image
image

File: src/main/webapp/app/exam/exam-scores/exam-scores.component.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
                 // Update the exerciseGroup statistic
                 const exGroupResult = groupIdToGroupResults.get(Number(exGroupId));
                 if (!exGroupResult) {
-                    // This should never been thrown. Indicates that the information in the ExamScoresDTO is inconsistent
+                    // This should never bee thrown. Indicates that the information in the ExamScoresDTO is inconsistent
```
Comment (Marvin-Vitcu) [inline src/main/webapp/app/exam/exam-scores/exam-scores.component.ts:298]:
```suggestion
                    // This should never be thrown. Indicates that the information in the ExamScoresDTO is inconsistent
```

File: src/main/webapp/app/exam/exam-scores/exam-scores.component.ts [hunk 2]
```diff
@@ -304,7 +304,7 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
                 // Update the specific exercise statistic
                 const exerciseResult = exGroupResult.exerciseResults.find((exResult) => exResult.exerciseId === studentExerciseResult.exerciseId);
                 if (!exerciseResult) {
-                    // This should never been thrown. Indicates that the information in the ExamScoresDTO is inconsistent
+                    // This should never bee thrown. Indicates that the information in the ExamScoresDTO is inconsistent
```
Comment (Marvin-Vitcu) [inline src/main/webapp/app/exam/exam-scores/exam-scores.component.ts:307]:
```suggestion
                    // This should never be thrown. Indicates that the information in the ExamScoresDTO is inconsistent
```

**Review by Marvin-Vitcu — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/exam/exam-scores/exam-scores.component.ts (hunk 2 above)
Comment (Strohgelaender) [reply src/main/webapp/app/exam/exam-scores/exam-scores.component.ts]: 🐝

**Review by bassner — APPROVED** [review]
Looks good to me, didn't find any more mistakes 👍

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/docs/user/exercises/programming-exercise-setup.inc#L386

repository is authoritative

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/commands/orderedListCommand.ts#L56

format

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/commands/underline.command.ts#L15

underline

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/commands/unorderedListCommand.ts#L50

format

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/overview/course-exams/course-exams.component.ts#L45

unsubscribe from all subscriptions? I suppose, I didn't quite understand what was meant by that

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/overview/course-statistics/course-statistics.component.ts#L577

corresponding

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/domain/FileUploadSubmission.java#L36

Past tense of split is split

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/domain/enumeration/ParticipationLifecycle.java#L6

participations

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/repository/ModelAssesmentKnowledgeRepository.java#L12

Assessment (will require a lot of refactoring because it's a name of a class)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/repository/ProgrammingExerciseTestCaseRepository.java#L42

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/repository/ProgrammingExerciseTestCaseRepository.java#L55

Associated

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/repository/TextAssesmentKnowledgeRepository.java#L12

Assessment

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/MailService.java#L77

Resource

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/SubmissionExportService.java#L61
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/SubmissionExportService.java#L74

Zipped

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/TextBlockService.java#L76
Submitted
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/TextBlockService.java#L116
Submission

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/TextExerciseImportService.java#L53
Assessment

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java#L542
Default

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java#L910
Separate

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L236
Regularly
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L252
Regularly
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L282
existent
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L297
existent

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L507
liveliness
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabService.java#L459
access

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsUserManagementService.java#L307

Assistant

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/jobs/JenkinsJobPermissionsUtils.java#L114

Permission

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java#L694
course

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/exam/ExamSessionService.java#L38

Added

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java#L253

Submissions

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/metis/PostService.java#L695

lowercase

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/plagiarism/PlagiarismService.java#L30
Anonymised
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/plagiarism/PlagiarismService.java#L55
Anonymised

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/plagiarism/ProgrammingPlagiarismDetectionService.java#L211
Plagiarism
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/plagiarism/ProgrammingPlagiarismDetectionService.java#L238
Directories

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseExportService.java#L552
Anonymise/Anonymisation

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseExportService.java#L429
Occurred

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionResultSimulationService.java#L65
participate

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionService.java#L648
correction round (2 words)

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionService.java#L652
submissions

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionService.java#L653
submissions

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingSubmissionService.java#L722
submissions

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/scheduled/AtheneScheduleService.java#L127

processing

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/service/scheduled/quiz/QuizScheduleService.java#L288
Registered???

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java#L479

submitterName

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java#L433

uploaded

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java#L118

split

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java#L417

Participations

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/StaticCodeAnalysisResource.java#L99

Past tense of reset is reset

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/java/de/tum/in/www1/artemis/web/rest/UserJWTController.java#L108

Caught

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/course/manage/detail/course-detail-doughnut-chart.component.ts#L101
Assigns

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/overview/discussion-section/discussion-section.component.html#L1

expanded

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/i18n/en/health.json#L23

Liveliness (liveness is not a word)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/i18n/en/lecture.json#L46

Title (I suppose it ought to be in English because it is under the "en" folder)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/entities/course.model.ts#L92
typecast
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/entities/course.model.ts#L94
typecast
past tense of typecast is typecast

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exam/exam-scores/exam-scores.component.html#L10
weight

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exam/exam-scores/exam-scores.component.ts#L856
recalculating

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/text/manage/example-text-submission/example-text-submission.component.ts#L143

everything

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/shared/exercise/exercise.service.ts#L20
Serviceable

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/shared/exercise/exercise.service.ts#L134
reset

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/quiz/manage/apollon-diagrams/exercise-generation/quiz-exercise-generator.ts#L238
absolute
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/quiz/manage/apollon-diagrams/exercise-generation/quiz-exercise-generator.ts#L239
percentage

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.ts#L117
Exercise
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.ts#L159
Exercise

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.ts#L83

handled

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts#L327
Tuple
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts#L356
Tuple
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts#L369
Tuple

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/admin/health/health.model.ts#L3

Liveliness

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/core/interceptor/errorhandler.interceptor.ts#L13
broadcast (past tense of broadcast is broadcast)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/core/interceptor/notification.interceptor.ts#L13
broadcast (past tense of broadcast is broadcast)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/programming/shared/code-editor/layout/code-editor-repository-is-locked.component.ts#L10
label
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/programming/shared/code-editor/layout/code-editor-repository-is-locked.component.ts#L18
label

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/lecture/lecture-import.component.ts#L140
delay

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/lecture/lecture.service.ts#L159
as well

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/course/manage/course-group.component.ts#L91

course

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/course/manage/course-management.component.ts#L117
expanded
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/course/manage/course-management.component.ts#L124
expanded

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.ts#L120
exercise

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/modeling/manage/modeling-exercise-import.component.html#L8
exercise

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/programming/manage/programming-exercise-import.component.html#L8
exercise

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/programming/manage/programming-exercise-import.component.ts#L111
element

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/text/manage/text-exercise-import.component.html#L8
exercise

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/text/manage/text-exercise-import.component.ts#L54
delay
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exercises/text/manage/text-exercise-import.component.ts#L85
element

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/markdown-editor.component.ts#L113

default

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/markdown-editor.component.ts#L293

split

https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/markdown-editor.component.ts#L328

whitespace

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/shared/markdown-editor/markdown-editor.component.ts#L89

colour picker (2 words)

**Comment by babyygemperor** [comment]
https://github.com/ls1intum/Artemis/blob/fbc2f20a433d0d16daf00b606a952df3d24bc75d/src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.ts#L96

submission

**Comment by babyygemperor** [comment]
Okay I'll make my own pull request for the rest of the typos, pointing out all the other typos outside your code changes doesn't make sense.

**Review by SimonPaul02 — APPROVED** [review]
Reviewed code together with @yuling-sun . Mostly all corrections were correct, but we found some additional changes:
In our opinion, there should by a hyphen after "non" -> non-instructors
image
Also, sometimes "ID", "Id" and "id" are used, there is no strict convention. 
Furthermore, we aren't sure what that comment wants to say:
image
Additionally, we think that the following change is incorrect: (should be changes occur or a change occurs)
image
And we found an error in this change: ("is" should be deleted)
image

**Review by babyygemperor — APPROVED** [review]
Looks good

**Comment by Strohgelaender** [comment]
Thank you for the review @SimonPaul02 and @yuling-sun, I fixed most of the points meantioned.

> Also, sometimes "ID", "Id" and "id" are used, there is no strict convention.

I noticed that, but since I also don't know what version should be prefered here, I just oped to keep it.

> we found a couple of comma-mistakes

I also found a few of them, but decided to fix them only in the documentation. I feel like it's almost cosistently not used in comments, so I just opted to keep that.

**Review by Dominik-Weinzierl — DISMISSED** [review]
I scrolled through your changes and it looks good to me :tada:

**Review by ge65cer — DISMISSED** [review]

**Review by nadinesamyy — APPROVED** [review]
checked commit changes, look good

**Review by ge65cer — APPROVED** [review]

**Review by maximiliansoelch — DISMISSED** [review]

**Review by ge65cer — APPROVED** [review]
Let's get this merged, so you don't have to update it all the time.

**Pull request merged**